sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/SclService.java

```diff
@@ -392,44 +406,42 @@ public static SclRootAdapter importSTDElementsInSCD(@NonNull SclRootAdapter scdR
 
 
         // List all Private and remove duplicated one with same iedName
-        Map<String, TPrivate> tPrivateMap = createMapIEDNameAndPrivate(scdRootAdapter);
+        Map<String, TPrivate> mapIEDNameAndPrivate = createMapIEDNameAndPrivate(scdRootAdapter);
 
-        Map<String, Pair<TPrivate, List<SCL>>> stringListMap = createMapICDSystemVersionUuidAndSTDFile(stds);
-        for (Map.Entry<String, Pair<TPrivate, List<SCL>>> entry : stringListMap.entrySet()) {
-            if (entry.getValue().getRight().size() != 1){
+        Map<String, Pair<TPrivate, List<SCL>>> mapICDSystemVersionUuidAndSTDFile = createMapICDSystemVersionUuidAndSTDFile(stds);
+        for (Map.Entry<String, Pair<TPrivate, List<SCL>>> entry : mapICDSystemVersionUuidAndSTDFile.entrySet()) {
```

Comment:
It seems that `entry.getKey()` is never used. 
Why not iterate directly on `mapICDSystemVersionUuidAndSTDFile.values()` then (instead of `entrySet`) ?